BeeTracker/app/src/main/res/layout/review_fragment.xml

```diff
@@ -1,19 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-        android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#F5F5F5">
+<RelativeLayout
+android:layout_width="match_parent"
+android:layout_height="wrap_content">
+<Button
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/back"
+    android:gravity="center_vertical"
+    android:layout_alignParentLeft="true"
+    android:layout_marginTop="8dp"
+    android:layout_marginBottom="8dp"
+    android:layout_marginLeft="8dp"
+    android:id="@+id/backButton"/>
+<TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_centerInParent="true"
+    android:gravity="center"
+    android:text="@string/step_three"/>
+<Button
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/submit"
+    android:gravity="center_vertical"
+    android:layout_alignParentRight="true"
+    android:layout_marginTop="8dp"
+    android:layout_marginBottom="8dp"
+    android:layout_marginRight="8dp"
+    android:id="@+id/submitButton"/>
+</RelativeLayout>
+    <LinearLayout
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Review fragment"/>
-    <Button
-        android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:layout_marginTop="30sp"
+        android:layout_marginLeft="20sp">
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@mipmap/ic_bumblebee"/>
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="IDENTIFIED SPECIES"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textStyle="bold|italic"
+                android:text="@string/speciesID"
+                android:id="@+id/speciesID"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/scientificID"
+                android:id="@+id/scientificID"/>
+        </LinearLayout>
+    </LinearLayout>
+    <LinearLayout
```

Comment:
Do you need this Linear layout? the textview is just stacked vertically like the other components, so it probably doesn't need to be wrapped.

Nesting a bunch of Linear Layouts can cause performance issues. Avoiding Linear Layouts when you don't need them is good, as is using Relative Layout instead. I think this screen definitely needs Linear Layouts, but fewer than how many you have here.